### PR TITLE
e2e test update for stabilization.

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
MatrixPage page object locators update.
FamilyPage constructor update for stability.
Disable test related to #1311

### CODE REVIEW

File: test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { browser } from 'protractor';
+import {browser, protractor} from 'protractor';
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts]: why do you need "protractor" here?

File: test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { browser } from 'protractor';
+import {browser} from 'protractor';
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts]: WEbstormSettings => Editor => CodeStyle => TypeScript => Spaces => ES^ import/export braces - should be checked

File: test-e2e/app/Tests/MatrixPage/MatrixPageZoomBtn.e2e-spec.ts [hunk 3]
```diff
@@ -1,21 +1,24 @@
 import { browser } from 'protractor';
 
 import { MatrixPage } from '../../Pages';
+import {scrollIntoView} from "../../Helpers/commonHelper";
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/MatrixPage/MatrixPageZoomBtn.e2e-spec.ts]: why do you need this import here?

File: test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts (hunk 1 above)
Comment (AntonOlkhovskyi) [reply test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts]: Debug leftover. Deleted.

File: test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts (hunk 2 above)
Comment (AntonOlkhovskyi) [reply test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts]: Done.

File: test-e2e/app/Tests/MatrixPage/MatrixPageZoomBtn.e2e-spec.ts (hunk 3 above)
Comment (AntonOlkhovskyi) [reply test-e2e/app/Tests/MatrixPage/MatrixPageZoomBtn.e2e-spec.ts]: Debug leftover. Deleted.

File: test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { browser } from 'protractor';
 import { getRandomNumber, isInViewport } from '../../Helpers';
 import { MatrixPage, CountryPage, AbstractPage, FamilyPage } from '../../Pages';
 import { MatrixImagePreview, FamilyImage, WelcomeWizard, FamilyImagePreview } from '../../Pages/Components';
+import {scrollIntoView} from '../../Helpers/commonHelper';
```
Comment (ludmilanesvitiy) [inline test-e2e/app/Tests/MatrixPage/MatrixImagePreview.e2e-spec.ts:6]: code => reformat Code

**Pull request merged**